### PR TITLE
Update README

### LINKED ISSUE #516: Add badge to RTD on the README

### PR DESCRIPTION
**Pull request by alexfoias**
Added the RTD badge.

Fixes #516

### CODE REVIEW

**Review by lrouhier — APPROVED** [review]
Thanks !

File: README.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
 [![Coverage Status](https://coveralls.io/repos/github/ivadomed/ivadomed/badge.svg?branch=master)](https://coveralls.io/github/ivadomed/ivadomed?branch=master)
 ![](https://github.com/neuropoly/ivadomed/workflows/Python%20package/badge.svg)
+[![Documentation Status](https://readthedocs.org/projects/ivado-medical-imaging/badge/?version=latest)](https://ivadomed.org/en/latest/?badge=latest)
```
Comment (jcohenadad) [inline README.md:6]: why "?badge=latest" in the URL?

Comment (alexfoias) [reply README.md:6]: sorry just took the default setting for the badge. Do you want to use stable ?

Comment (jcohenadad) [reply README.md:6]: i think so, yes.

Comment (alexfoias) [reply README.md]: fixed in 6641660

**Pull request merged**